### DISCUSSION TITLE
Get the health healed in `onHeal` of `HealthComp`

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#692] [Ideas]
Would be cool if we can have the healed health in the `onHeal` event, such like:

```js
onHeal((heal) => {
    if(heal> 100) k.play("yupi")
})
```

And will be cool if there's the same option for `onHurt`.

**Comment by slmjkdbtl** [comment]
Added in a8096204a48bd3311ab0543e5d6ecd166c629202
